sql/telemetry_derived/clients_last_seen_v1/init.sql

```diff
@@ -3,22 +3,21 @@ CREATE TABLE
 PARTITION BY
   submission_date
 CLUSTER BY
-  sample_id,
-  client_id AS
+  normalized_channel,
+  sample_id
 OPTIONS
-  ( require_partition_filter=TRUE) AS
+  (require_partition_filter=TRUE) AS
 SELECT
-  CAST(NULL AS DATE) AS submission_date,
-  0 AS days_seen_bits,
-  0 AS days_visited_5_uri_bits,
-  0 AS days_opened_dev_tools_bits,
-  0 AS days_created_profile_bits,
-  [STRUCT("" AS experiment, "" AS branch, 0 AS bits)] AS days_seen_in_experiment,
+  NULL AS days_seen_bits,
```

Comment:
The zeroes here ensured we got the right type for the field. I believe INT64 is the default type used for NULLs, but I'd rather not rely on that implicit behavior. We can make these `CAST(NULL AS INT64)` to be explicit.